cmd/broker/main.go

```diff
@@ -64,14 +70,21 @@ func runWithContext(ctx context.Context) error {
 		return nil
 	}
 
+	namespace := os.Getenv("POD_NAMESPACE")
+
 	addr := ":" + strconv.Itoa(options.Port)
 	var err error
 
-	sharedResourceClient, err := getSharedResourceClient()
+	sharedResourceClient, err := getSharedResourceClient(namespace)
 	if err != nil {
 		return err
 	}
-	ctrlr := controller.CreateController(sharedResourceClient)
+	sharedServiceSliceClient, err := getSharedServiceSliceResourceClient(namespace)
+	if err != nil{
+		return err
+	}
+	ctrlr := controller.CreateController(namespace,sharedResourceClient,sharedServiceSliceClient)
+	ctrlr.Catalog()
```

Comment:
Do you need to call `Catalog` here?

Reply:
It is basically a bootstrap to populate the services data